### PR TITLE
Fix a_jazz/akc084 render

### PR DESCRIPTION
**Pull request by plodah**
## Description

Both the ASD and ZXC rows on a_jazz/akc084 have y: value of 3 in LAYOUT.
Keys render on top of each other in config.qmk.fm.
image

To fix this, ZXC and bottom rows Y value incremented.

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by plodah** [comment]
Output of `qmk info -kb a_jazz/akc084 -km default -m`

```
BEFORE
┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐
│0A││0B││0C││0D││0E││0F││0G││0H││0I││0J││0K││0L││0M││0N││3N││4M││6M│
└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘
┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──────┐┌──┐
│1A││1B││1C││1D││1E││1F││1G││1H││1I││1J││1K││1L││1M││1N    ││4N│
└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──────┘└──┘
┌────┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐    ┌────┐┌──┐
│2A  ││2B││2C││2D││2E││2F││2G││2H││2I││2J││2K││2M│    │2N  ││5L│
└────┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘    └────┘└──┘
┌───────┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌─────┐┌──┐┌──┐
│4A     ││4B││4C││4D││4E││4F││4G││4H││4I││4J││4K││4L   ││5H││5N│
└───────┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└─────┘└──┘└──┘
┌───┐┌───┐┌───┐┌───────────────────────┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐
│5A ││5B ││5C ││5D                     ││5E││5F││5G││5I││5J││5K│
└───┘└───┘└───┘└───────────────────────┘└──┘└──┘└──┘└──┘└──┘└──┘

AFTER
┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐
│0A││0B││0C││0D││0E││0F││0G││0H││0I││0J││0K││0L││0M││0N││3N││4M││6M│
└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘
┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──────┐┌──┐
│1A││1B││1C││1D││1E││1F││1G││1H││1I││1J││1K││1L││1M││1N    ││4N│
└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──────┘└──┘
┌────┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐    ┌────┐┌──┐
│2A  ││2B││2C││2D││2E││2F││2G││2H││2I││2J││2K││2M│    │2N  ││5L│
└────┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘    └────┘└──┘
┌─────┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌───────┐┌──┐┌──┐
│3A   ││3B││3C││3D││3E││3F││3G││3H││3I││3J││3K││3L     ││3M││5M│
└─────┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└───────┘└──┘└──┘
┌───────┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌─────┐┌──┐┌──┐
│4A     ││4B││4C││4D││4E││4F││4G││4H││4I││4J││4K││4L   ││5H││5N│
└───────┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└─────┘└──┘└──┘
┌───┐┌───┐┌───┐┌───────────────────────┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐
│5A ││5B ││5C ││5D                     ││5E││5F││5G││5I││5J││5K│
└───┘└───┘└───┘└───────────────────────┘└──┘└──┘└──┘└──┘└──┘└──┘
```

File: keyboards/a_jazz/akc084/keyboard.json [hunk 1]
```diff
@@ -98,30 +98,30 @@
                 {"matrix": [3, 11], "x": 11.75, "y": 3, "w": 2.25},
                 {"matrix": [3, 12], "x": 14, "y": 3},
```
Comment (noroadsleft) [inline keyboards/a_jazz/akc084/keyboard.json:99]:
```suggestion
                {"matrix": [3, 11], "x": 11.75, "y": 3},
                {"matrix": [3, 12], "x": 12.75, "y": 3, "w": 2.25},
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
Since you're here, if you could add the following changes to this PR, it would save some additional PR fixes later.

image

1. Change Line 84 to fix the two square bracket keys being in the same place:
   ```
                   {"matrix": [2, 12], "x": 12.5, "y": 2},
   ```
2. Apply the GitHub suggestion below to fix the key sizes/positioning for the Quote and Enter keys.

**Review by zvecr — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]
Looks good, thanks!

image

**Pull request merged**